frontend/src/metabase/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,58 @@
+import type { ReactNode } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+
+import { Button, Flex, Modal, Text } from "metabase/ui";
+
+interface ConfirmContentProps {
+  opened: boolean | undefined;
+  "data-testid"?: string;
+  title: string | ReactNode;
+  content?: string | null;
+  message?: string | ReactNode;
+  onClose?: () => void;
+  onConfirm?: () => void;
+  confirmButtonText?: string;
+  confirmButtonPrimary?: boolean;
+}
+
+export const ConfirmationModal = ({
+  opened,
+  "data-testid": dataTestId,
+  title,
+  content = null,
+  message = t`Are you sure you want to do this?`,
+  onClose = _.noop,
+  onConfirm = _.noop,
+  confirmButtonText = t`Yes`,
```

Comment:
We previously had `cancelButtonText` that defaulted to `Cancel`, but I couldn't find anywhere the default wasn't used